logrlint.go

```diff
@@ -26,6 +26,7 @@ func newAnalyzer() *analysis.Analyzer {
 var validLoggerToFuncNames = map[string]stringSet{
 	"github.com/go-logr/logr": newStringSet([]string{"Error", "Info", "WithValues"}),
 	"k8s.io/klog/v2":          newStringSet([]string{"InfoS", "InfoSDepth", "ErrorS"}),
+	"go.uber.org/zap":         newStringSet([]string{"Fatalw", "Panicw", "DPanicw", "Errorw", "Warnw", "Infow", "Debugw", "With"}),
```

Comment:
There are two `With`, one is https://pkg.go.dev/go.uber.org/zap#Logger.With.

It will cause false-positive for this case